src/components/widgets/camera/CameraItem.vue

```diff
@@ -1,67 +1,66 @@
 <template>
-  <div>
-    <v-sheet
-      :elevation="0"
-      rounded
-      class="camera-container"
-      v-on="$listeners"
+  <v-sheet
+    :elevation="0"
+    rounded
+    class="camera-container"
+    v-on="$listeners"
+  >
+    <img
+      v-if="camera.type === 'mjpgstream' || camera.type === 'mjpgadaptive'"
+      ref="camera_image"
+      :src="cameraUrl"
+      class="camera-image"
+      @load="handleImgLoad"
     >
-      <img
-        v-if="camera.type === 'mjpgstream' || camera.type === 'mjpgadaptive'"
-        ref="camera_image"
-        :src="cameraUrl"
-        class="camera-image"
-        @load="handleImgLoad"
-      >
 
-      <video
-        v-if="camera.type === 'ipstream'"
-        ref="camera_image"
-        :src="cameraUrl"
-        autoplay
-        class="camera-image"
-      />
-
-      <iframe
-        v-if="camera.type === 'iframe'"
-        ref="camera_image"
-        :src="cameraUrl"
-        class="camera-image"
-        :height="cameraHeight"
-        frameBorder="0"
-      />
-
-      <div
-        v-if="camera.name"
-        class="camera-name"
-      >
-        {{ camera.name }}
-      </div>
-      <div
-        v-if="camera.type === 'mjpgadaptive' && time"
-        class="camera-frames"
-      >
-        fps: {{ currentFPS }}
-      </div>
-      <div
-        v-if="cameraFullScreenUrl"
-        class="camera-fullscreen"
+    <video
+      v-if="camera.type === 'ipstream'"
+      ref="camera_image"
+      :src="cameraUrl"
+      autoplay
+      class="camera-image"
+    />
+
+    <iframe
+      v-if="camera.type === 'iframe'"
+      ref="camera_image"
+      :src="cameraUrl"
+      class="camera-image"
+      :height="cameraHeight"
+      frameBorder="0"
+    />
+
+    <div
+      v-if="camera.name"
+      class="camera-name"
+    >
+      {{ camera.name }}
+    </div>
+    <div
+      v-if="camera.type === 'mjpgadaptive' && time"
+      class="camera-frames"
+    >
+      fps: {{ currentFPS }}
+    </div>
+    <div
+      v-if="cameraFullScreenUrl && (!fullscreen || fullscreenMode === 'embed')"
+      class="camera-fullscreen"
+    >
+      <a
+        :href="cameraFullScreenUrl"
+        :target="fullscreenMode === 'rawstream' ? '_blank' : ''"
```

Comment:
```suggestion
        :target="fullscreen || fullscreenMode === 'rawstream' ? '_blank' : ''"
```

This bit also needs to check for `fullscreen`, because without that check it will just replace Fluidd page with the camera